ext/tiny_tds/result.c

```diff
@@ -99,12 +99,14 @@ static void nogvl_cleanup(DBPROCESS *client) {
   Now that the blocking operation is done, we can finally throw any
   exceptions based on errors from SQL Server.
   */
-  for (short int i = 0; i < userdata->nonblocking_errors_length; i++) {
+  short int i;
+  short int j;
+  for (i = 0; i < userdata->nonblocking_errors_length; i++) {
     tinytds_errordata error = userdata->nonblocking_errors[i];
 
     // lookahead to drain any info messages ahead of raising error
     if (!error.is_message) {
-      for (short int j = i; j < userdata->nonblocking_errors_length; j++) {
+      for (j = i; j < userdata->nonblocking_errors_length; j++) {
```

Comment:
Is the idea here to declare `j` outside of the `i` loop so as to only allocate once instead of allocating `i` times?

Reply:
Actually that wasn't my intention. I should probably move the `j` declaration inside the `i` loop.